CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Show the region on the project summary and school details.
 - Notes can be edited by the user who created them
 - Notes can be deleted by the user who created them
+- Added richer documentation for the workflow schemas
```

Comment:
Is there a way to move the docs into the `doc` directory - I feel like that is the consistent place for all documentaiton which this breaks?